CODEOWNER

```diff
@@ -6,4 +6,4 @@
 
 # The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
 
-* @derberg @magicmatatjahu @asyncapi-bot-eve
+* @derberg @magicmatatjahu @asyncapi-bot-eve @Shurtu-gal
```

Comment:
@magicmatatjahu pinging you so you are aware

@Shurtu-gal completely rewrites this action and I proposed to him to be maintainer after we merge

are you ok, and also are you still interested to be a maintainer here?

Reply:
I can be still to accept PRs when they are very important :) @Shurtu-gal welcome!

Reply:
Thanks